feature-libs/quote/components/links/quote-links.component.html

```diff
@@ -20,6 +20,24 @@
           {{ 'quote.links.quotes' | cxTranslate }}
         </a>
       </li>
+      <li>
+        <button
+          type="button"
+          class="link cx-static-action-link"
+          *ngIf="
+            hasAttachment(quoteDetails.sapAttachments!) &&
```

Comment:
hasAttachment(quoteDetails.sapAttachments**!**)
@suprishi, I am not familiar with this syntax. Could you explain to me?

Reply:
The `!` at the end acts as a null-assertion operator. It tells VSCode/compiler that `sapAttachments` will not be a null.

Reply:
Okay!, thanks for explaining  to me.